src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -6579,14 +6619,18 @@ subroutine atm_bdy_adjust_dynamics_relaxzone_tend( config, tend, state, diag, me
          end if
 
       end do
+      !$acc end parallel
 
       !  Third (and last), the horizontal filter for ru
-
+      !$acc parallel default(present)
+      !$acc loop gang worker private(cell1, cell2, vertex1, vertex2, r_dc, r_dv, & 
+      !$acc    iCell, iVertex, invArea, iEdge_div, iEdge_vort, edge_sign, &
+      !$acc    laplacian_filter_coef, divergence1, divergence2, vorticity1, vorticity2)
```

Comment:
Here I'd only expect the arrays (`divergence1`, `divergence2`, `vorticity1`, and `vorticity2`) to be declared private. The rest should automatically be private at the worker level and passed on to their (vector) threads. Are you being explicit here, or do you need to also add the scalar-values to the private clause for correctness?

Reply:
True. I briefly didn't remember that about scalars when I wasn't getting bit identical results. Will make these changes and try again.